_tools/terraform.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: default
+title: Terraform
+nav_order: 180
+has_children: false
+---
+
+# Terraform provider
+
+You can use the Terraform OpenSearch provider to provision OpenSearch resources and interact with the OpenSearch API. The Terraform provider supports AWS OpenSearch Service domains and OpenSearch clusters deployed on Kubernetes and other infrastructure. For more information, see the [Terraform provider documentation](https://registry.terraform.io/providers/opensearch-project/opensearch/latest/docs).
```

Comment:
"other infrastructures" (plural)?

Reply:
I think it makes sense to use "infrastructure" as a plural noun here.